src/main/java/supercoder79/simplexterrain/mixin/MixinLevelGeneratorTypeClient.java

```diff
@@ -1,32 +0,0 @@
-package supercoder79.simplexterrain.mixin;
-
```

Comment:
This is still necessary

Reply:
Wait i think this might be the wrong mixin lemme check. Supercoder changed some of the code

Reply:
Nope it’s the server mixin

Reply:
Actually you have some server properties mixins